Formula/file-roller.rb

```diff
@@ -11,30 +11,48 @@ class FileRoller < Formula
     sha256 "05ca1ccdd5771e574bf4b1bad58fa0ddc6d2db65d2fda7999912ced5e0cb704e" => :mavericks
   end
 
-  depends_on "pkg-config" => :build
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
   depends_on "intltool" => :build
   depends_on "itstool" => :build
+  depends_on "libtool" => :build
   depends_on "libxml2" => ["with-python", :build]
+  depends_on "pkg-config" => :build
+  depends_on "yelp-tools" => :build
+
+  depends_on "atk"
+  depends_on "cairo"
+  depends_on "gdk-pixbuf"
+  depends_on "gettext"
+  depends_on "glib"
+  depends_on "gnome-icon-theme"
   depends_on "gtk+3"
+  depends_on "hicolor-icon-theme"
   depends_on "json-glib"
-  depends_on "libmagic"
   depends_on "libarchive"
-  depends_on "hicolor-icon-theme"
-  depends_on "gnome-icon-theme"
+  depends_on "libmagic"
+  depends_on "pango"
```

Comment:
Apart from sorting things, there are a lot of new dependencies here and in the build dependencies above. How come? This change is neither mentioned in the PR body nor in the commit message.